i18n/qgis_fr.ts

```diff
@@ -11133,12 +11134,12 @@ Maintenir Ctrl (Cmd) incrémente la rotation de 15 deg.</translation>
     <message>
         <location filename="../src/ui/qgisapp.ui" line="2748"/>
         <source>&amp;Layout Extents…</source>
-        <translation>&amp;Étendue de mise en page</translation>
+        <translation>&amp;Emprises de mise en page…</translation>
     </message>
     <message>
         <location filename="../src/ui/qgisapp.ui" line="2751"/>
         <source>Layout Extents</source>
-        <translation>Étendue de la mise en page</translation>
+        <translation>Emprise de la mise en page</translation>
```

Comment:
I'm not sure but `Emprise` -> `EmpriseS` for consistency with the case above?

Reply:
Aaaarrrghhhh!!! Another round of update 😠